### PR TITLE
chore: spfx help doc 

### PR DESCRIPTION
**Pull request by yuqizhou77**

### CODE REVIEW

File: docs/fx-core/spfx-help-v5.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
 # SPFx troubleshoot
```
Comment (summzhan) [inline docs/fx-core/spfx-help-v5.md:1]: SPFx FAQ and troubleshooting guide

File: docs/fx-core/spfx-help-v5.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
 
+1. [Failure to scaffold](#scaffold)
```
Comment (summzhan) [inline docs/fx-core/spfx-help-v5.md:3]: Scaffolding errors / Scaffolding failures / Failed to scaffold. Either one could work.

**Review by summzhan — CHANGES_REQUESTED** [review]
only title and wording change

File: docs/fx-core/spfx-help-v5.md (hunk 1 above)
Comment (yuqizhou77) [reply docs/fx-core/spfx-help-v5.md]: updated.

File: docs/fx-core/spfx-help-v5.md (hunk 2 above)
Comment (yuqizhou77) [reply docs/fx-core/spfx-help-v5.md]: updated.

**Review by summzhan — DISMISSED** [review]

**Review by summzhan — APPROVED** [review]

**Pull request merged**